src/Tribe/Commerce/PayPal/Main.php

```diff
@@ -987,7 +987,10 @@ public function generate_tickets( $payment_status = 'completed', $redirect = tru
 	/**
 	 * Sends ticket email
 	 *
+	 * @since TBD added $post_id parameter
+	 *
 	 * @param int $order_id Order post ID
+	 * @param int $post_id  Parent post ID
 	 */
 	public function send_tickets_email( $order_id ) {
```

Comment:
We need receiving any second param here. Make sure we make it optional, since older code will break otherwise.